index.html

```diff
@@ -349,6 +343,41 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>If the element, which is set as the target for mouse events while the pointer is
+           locked, is a
+           <a href="https://dom.spec.whatwg.org/#concept-shadow-including-descendant">shadow-including descendant</a> of the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+           return the result of the following algorithm:
+           <ol>
+             <li>Let <var>target</var> be the target for mouse events while the pointer is locked.
+             <li>If <var>target</var> is in the same tree as the
+               <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+               let <var>adjusted-target</var> be <var>target</var></li>
+             <li>Otherwise: let <var>adjusted-target</var> be the shadow host which is a
```

Comment:
I thought that 

- there should be one shadow host that matches the conditions, so "THE shadow host"
- which meets both of the following conditions (AND condition):
    - "A shadow-including ancestor" of target (because there can be multiple ancestor shadow hosts), and
    - is "in THE same tree" as the context object (this should filter only one in the same tree)

therefore I can use "THE shadow host" here.
I'll ask some native speaker around.